packages/cli/src/lib/onboarding/local.test.ts

```diff
@@ -177,6 +177,71 @@ describe('local templates', () => {
     });
   });
 
+  describe('styling libraries', () => {
```

Comment:
These tests were here for the classic remix compiler but we removed them when migrating to Vite without styling support. I've copied them back and adapted them to Vite.